samlsp/middleware.go

```diff
@@ -1,14 +1,15 @@
 package samlsp
 
 import (
-	"crypto/x509"
 	"encoding/base64"
 	"encoding/xml"
 	"fmt"
 	"net/http"
 	"strings"
 	"time"
 
+	"crypto"
```

Comment:
nit: can we fix up the imports here, please

Reply:
I will update PR, thanks for the feedback